src/core/annotation.js

```diff
@@ -139,6 +141,70 @@ var Annotation = (function AnnotationClosure() {
 
   Annotation.prototype = {
     /**
+     * Set the flags.
+     *
+     * @public
+     * @memberof Annotation
+     * @param {number} flags - unsigned 32-bit integer specifying annotation
+     *                         characteristics
+     * @see {@link shared/util.js}
+     */
+    setFlags: function Annotation_setFlags(flags) {
+      if (flags === (flags | 0)) {
```

Comment:
Nit: it might be a *tiny* bit simpler to understand this line if [`isInt`](https://github.com/mozilla/pdf.js/blob/master/src/shared/util.js#L1018) is used here :-)